service/eventer/github/client.go

```diff
@@ -23,7 +24,7 @@ const (
 	// DeploymentUrlFormat is the format string used to compute the Github
 	// API URL used to fetch the latest deployment event for a specific
 	// environment.
-	DeploymentUrlFormat = "https://api.github.com/repos/%s/%s/deployments?environment=%s"
+	DeploymentUrlFormat = "https://api.github.com/repos/%s/%s/deployments"
```

Comment:
The plain format parsing failed when I used my playground env `xh3b4sd+minikube`.